src/PostgREST/Logger.hs

```diff
@@ -56,15 +55,14 @@ logWithDebounce loggerState action = do
       newDebouncer
 
 -- TODO stop using this middleware to reuse the same "observer" pattern for all our logs
-middleware :: LogLevel -> (Wai.Request -> Maybe BS.ByteString) -> Wai.Middleware
-middleware logLevel getAuthRole =
+middleware :: LogLevel -> Wai.Middleware
+middleware logLevel =
     unsafePerformIO $
       Wai.mkRequestLogger Wai.defaultRequestLoggerSettings
       { Wai.outputFormat =
          Wai.ApacheWithSettings $
            Wai.defaultApacheSettings &
-           Wai.setApacheRequestFilter (\_ res -> shouldLogResponse logLevel $ Wai.responseStatus res) &
-           Wai.setApacheUserGetter getAuthRole
+           Wai.setApacheRequestFilter (\_ res -> shouldLogResponse logLevel $ Wai.responseStatus res)
```

Comment:
Now with this approach we can't get the role here, I think we should do this change as a separate `break/fix`?